tests/auth_tests/test_auth_backends.py

```diff
@@ -574,6 +578,20 @@ def test_changed_backend_settings(self):
             self.assertIsNotNone(user)
             self.assertTrue(user.is_anonymous)
 
+    @override_settings(AUTHENTICATION_BACKENDS=[
+        'auth_tests.test_auth_backends.SomeModelBackend',
+        'django.contrib.auth.backends.ModelBackend'])
+    def test_missing_get_user(self):
```

Comment:
This test is in the wrong location as you're not changing anything in `contrib.auth`. It should go in `test_client` or test_client_regress`, wherever `force_login()` is tested.